src/cmp-ui.test.ts

```diff
@@ -111,27 +119,36 @@ describe('cmp-ui', () => {
     });
 
     describe('canShow', () => {
-        it('canShow returns true if readGuCookie returns null', () => {
-            readIabCookie.mockReturnValue('foo');
-            readGuCookie.mockReturnValue(null);
-            expect(canShow()).toBe(true);
-        });
-
         it('canShow returns true if readIabCookie returns null', () => {
             readIabCookie.mockReturnValue(null);
-            readGuCookie.mockReturnValue('foo');
-            expect(canShow()).toBe(true);
-        });
-        it('canShow returns true if readIabCookie and readGuCookie return null', () => {
-            readIabCookie.mockReturnValue(null);
-            readGuCookie.mockReturnValue(null);
             expect(canShow()).toBe(true);
         });
 
-        it('canShow returns false if readIabCookie and readGuCookie return truthy values', () => {
+        it('canShow returns false if readIabCookie returns truthy value', () => {
             readIabCookie.mockReturnValue('foo');
-            readGuCookie.mockReturnValue('bar');
             expect(canShow()).toBe(false);
         });
+        // it('canShow returns true if readGuCookie returns null', () => {
```

Comment:
Are we missing these test cases now? As above, if they are redundant do they need to be here commented out?

Reply:
Linked to my comment https://github.com/guardian/consent-management-platform/pull/37/files#r332073439 these tests will be useful once we restore `readGuCookie()` to the `canShow` check.